resources/schema/MetaVariables.json

```diff
@@ -138,6 +138,12 @@
     },
     {
       "const": "variable_value_length"
+    },
+    {
+      "const": "define_vlm_mandatory"
+    },
+    {
+      "const": "define_variable_mandatory"
```

Comment:
Can you add these in alphabetical order. It make it easier to maintain.
You also need to add these to MetaVariables.md